Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -439,9 +444,12 @@ private void UpdateCurrentTextEdit(PretokenStringContainer edit)
             Text = edit.Text; // Update our text property.
         }
 
-        private async Task RemoveToken(TokenizingTextBoxItem item)
+        private async Task<bool> RemoveToken(TokenizingTextBoxItem item, object data = null)
```

Comment:
@marcpems what was the need for the extra parameter here?

Reply:
fixed